app/components/QuestionForm/index.js

```diff
@@ -116,7 +111,10 @@ export const QuestionForm = ({
 }) => {
   const [isSelectedType, setIsSelectedType] = useState(false);
   const [isError, setIsError] = useState(false);
+  const [isNotPressed, setIsNotPressed] = useState(true);
```

Comment:
Avoid using `Not` in variable names. It makes code harder to read.
This can be `const [submitPressed, setSubmitPressed] = useState(false);` and then you just reverse conditions.